addons/l10n_be_invoice_bba/models/account_invoice.py

```diff
@@ -20,14 +20,14 @@ class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
     @api.model
-    def _get_reference_type(self):
+    def _get_reference_types(self):
         """Add BBA Structured Communication Type and change labels from 'reference' into 'communication' """
-        res = super(AccountInvoice, self)._get_reference_type()
+        res = super(AccountInvoice, self)._get_reference_types()
         res[[i for i, x in enumerate(res) if x[0] == 'none'][0]] = ('none', _('Free Communication'))
```

Comment:
It seems that this line is not necessary anymore as you've already rename the 'none' selection to 'Free Communication' in account_invoice.py